### ISSUE TITLE
Performance of CIDEr decreases when performing self critic training at first 6000 iterations

### ISSUE THREAD

**Issue by vanpersie32** [ruotianluo/self-critical.pytorch#36]
Hi, I follow the instruction to train the model. Take the fc model for example. I train it with cross entropy loss for 25 epochs(iterations: 336000), and the CIDEr on validation set is 0.92. Then I further train the model with scst, its CIDEr on validation set is 0.89 at iterations 342000. I want to ask why the cider decrease
at the first 6000 iterations of scst(self critic sequence training).

**Comment by minizon** [comment]
Hi, have you worked out this problem? I encountered similar issue when training with SCST, the average training reward gradually rise from a negative number to 0, and the CIDEr on the evaluation set decrease from 1.1 to 0.9. I'm using the TopDown model, and when I decrease the batch size in SCST, the CIDEr decrease even faster than a larger batch size.

**Comment by kangkang59812** [comment]
same issue

**Comment by ruotianluo** [comment]
To be honest, that is really wierd. I have never seen this kind of behavior after I release this code. (I saw this only when I was implementing it.)
Are you using the same hyperparameter in the readme?

**Comment by kangkang59812** [comment]
I think we three meet a same problem. Did you @vanpersie32 @minizon  use pytorch 1.0 or pytorch1.1.?Maybe some functions has changed inside and this confused results occured when I trained faster_rcnn before. It's hard to locate it.

**Comment by atg93** [comment]
Hey, did you solve the problem ?
